container/providers.bzl

```diff
@@ -20,7 +20,7 @@ BundleInfo = provider(fields = ["container_images", "stamp"])
 FlattenInfo = provider()
 
 # A provider containing information exposed by container_image rules
-ImageInfo = provider(fields = ["container_parts"])
+ImageInfo = provider(fields = ["container_parts", "digest"])
```

Comment:
Would it make sense to make the digest part of the container_parts? If not, why?